src/inputs/buttons/text-button/PTextButton.vue

```diff
@@ -155,6 +160,10 @@ export default defineComponent<TextButtonProps>({
         cursor: not-allowed;
         text-decoration: none;
     }
+    &.readonly {
+        cursor: unset;
```

Comment:
suggestion: How about 'cursor: not-allowed;' like in the PButton component?

Reply:
Okay!